--- origsrc/kdepim-4.2.4/console/kabcclient/src/CMakeLists.txt	2008-05-29 06:19:37.000000000 -0500
+++ src/kdepim-4.2.4/console/kabcclient/src/CMakeLists.txt	2009-06-22 23:15:17.763279200 -0500
@@ -28,7 +28,7 @@ install(TARGETS kabcclient ${INSTALL_TAR
 
 if(NOT WIN32)
 #note: kabcclient acts as kabc2mutt invoked with that name
-install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/kabcclient DESTINATION ${BIN_INSTALL_DIR} RENAME kabc2mutt)
+install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/kabcclient${CMAKE_EXECUTABLE_SUFFIX) DESTINATION ${BIN_INSTALL_DIR} RENAME kabc2mutt${CMAKE_EXECUTABLE_SUFFIX})
 else(NOT WIN32)
 kde4_add_executable(kabc2mutt NOGUI ${kabcclient_SRCS})
 target_link_libraries(kabc2mutt ${KDE4_KABC_LIBS} ${KDE4_KDEUI_LIBS} )
