--- origsrc/kdepim-4.6.0/mobile/lib/calendar/CMakeLists.txt	2011-06-06 17:19:52.000000000 -0500
+++ src/kdepim-4.6.0/mobile/lib/calendar/CMakeLists.txt	2011-06-21 12:51:10.907577300 -0500
@@ -26,11 +26,13 @@ add_definitions( -DKDEPIMMOBILEUICALENDA
 kde4_add_library(kdepimmobileuicalendar STATIC ${mobileui_calendar_SRCS})
 
 target_link_libraries(kdepimmobileuicalendar
+  ${QT_QTDECLARATIVE_LIBRARY}
   ${KDEPIMLIBS_AKONADI_LIBS}
   ${KDEPIMLIBS_KCALCORE_LIBS}
   akonadi_next
   incidenceeditorsngmobile
   calendarsupport
+  kdepimmobileui
 )
 
 install(FILES incidence-editor.qml DESTINATION ${DATA_INSTALL_DIR}/mobileui)
