--- origsrc/kdepim-3.5.9/akregator/src/Makefile.am	2006-10-01 12:30:25.000000000 -0500
+++ src/kdepim-3.5.9/akregator/src/Makefile.am	2008-05-02 10:51:13.500000000 -0500
@@ -145,8 +145,8 @@
 	kspeechsink.skel \
 	kspeech.stub
 
-libakregatorpart_la_LDFLAGS = $(KDE_RPATH) $(KDE_PLUGIN) $(LIB_KUTILS) -avoid-version -no-undefined $(all_libraries)
-libakregatorpart_la_LIBADD = libakregatorprivate.la 
+libakregatorpart_la_LDFLAGS = $(KDE_PLUGIN) $(all_libraries)
+libakregatorpart_la_LIBADD = libakregatorprivate.la $(LIB_KUTILS)
 
 # this is where the desktop file will go
 partdesktopdir = $(kde_servicesdir)
--- origsrc/kdepim-3.5.9/kmail/Makefile.am	2008-02-13 03:39:37.000000000 -0600
+++ src/kdepim-3.5.9/kmail/Makefile.am	2008-05-02 10:51:13.515625000 -0500
@@ -44,6 +44,7 @@
 
 libkmail_bodypartformatter_application_octetstream_la_SOURCES = app_octetstream.cpp
 libkmail_bodypartformatter_application_octetstream_la_LDFLAGS = $(all_libraries) -module $(KDE_PLUGIN)
+libkmail_bodypartformatter_application_octetstream_la_LIBADD = $(LIB_KDECORE)
 
 linkdir = $(kde_datadir)/kmail/plugins/bodypartformatter
 link_DATA = application_octetstream.desktop
--- origsrc/kdepim-3.5.9/kontact/src/Makefile.am	2008-02-13 03:39:52.000000000 -0600
+++ src/kdepim-3.5.9/kontact/src/Makefile.am	2008-05-02 10:50:23.406250000 -0500
@@ -3,9 +3,9 @@
 INCLUDES = -I$(top_srcdir)/kontact/interfaces -I$(top_srcdir) $(all_includes)
 
 lib_LTLIBRARIES = libkontact.la
-libkontact_la_LDFLAGS = $(all_libraries) $(KDE_RPATH) -version-info 1:0
+libkontact_la_LDFLAGS = $(all_libraries) $(KDE_RPATH) -version-info 1:0 -no-undefined
 libkontact_la_LIBADD = $(LIB_KDECORE)
-libkontact_la_SOURCES = prefs.kcfgc
+libkontact_la_SOURCES = prefs.kcfgc dummy.cpp
 
 bin_PROGRAMS = kontact
 
@@ -40,3 +40,6 @@
 kontactsetdlg_DATA = kontact.setdlg
 
 KDE_ICON = AUTO
+
+dummy.cpp:
+	echo > $@
--- origsrc/kdepim-3.5.9/korganizer/Makefile.am	2008-02-13 03:39:54.000000000 -0600
+++ src/kdepim-3.5.9/korganizer/Makefile.am	2008-05-02 10:51:13.546875000 -0500
@@ -49,7 +49,7 @@
 lib_LTLIBRARIES = libkorganizer_eventviewer.la \
                   libkorganizer_calendar.la libkorganizer.la
 
-libkorganizer_la_LDFLAGS = $(all_libraries) $(KDE_RPATH) -version-info 1:0
+libkorganizer_la_LDFLAGS = $(all_libraries) $(KDE_RPATH) -version-info 1:0 -no-undefined
 libkorganizer_la_LIBADD  = \
 	$(top_builddir)/libkcal/libkcal.la \
 	$(top_builddir)/libkpimidentities/libkpimidentities.la \
--- origsrc/kdepim-3.5.9/kresources/groupware/kcal_resourcegroupware.h	2005-09-10 03:23:47.000000000 -0500
+++ src/kdepim-3.5.9/kresources/groupware/kcal_resourcegroupware.h	2008-05-02 10:51:13.562500000 -0500
@@ -29,9 +29,13 @@
 #include <kio/job.h>
 #include <kconfig.h>
 
+#include "kcal_groupwareprefsbase.h"
+
+/*
 namespace KPIM {
 class GroupwarePrefsBase;
 }
+*/
 
 namespace KCal {
 
--- origsrc/kdepim-3.5.9/kresources/slox/Makefile.am	2005-09-10 03:23:46.000000000 -0500
+++ src/kdepim-3.5.9/kresources/slox/Makefile.am	2008-05-02 10:51:13.578125000 -0500
@@ -6,18 +6,18 @@
 libkslox_la_SOURCES = sloxaccounts.cpp webdavhandler.cpp \
                       sloxfolder.cpp sloxfoldermanager.cpp \
                       sloxfolderdialog.cpp sloxbase.cpp
-libkslox_la_LDFLAGS = $(all_libraries)
+libkslox_la_LDFLAGS = $(all_libraries) -no-undefined
 libkslox_la_LIBADD  = $(top_builddir)/libkcal/libkcal.la \
                       $(top_builddir)/libkdepim/libkdepim.la
 
 libkcal_slox_la_SOURCES = kcalresourceslox.cpp kcalresourcesloxconfig.cpp \
                           kcalsloxprefs.kcfgc
-libkcal_slox_la_LDFLAGS = $(all_libraries)
+libkcal_slox_la_LDFLAGS = $(all_libraries) -no-undefined
 libkcal_slox_la_LIBADD  = libkslox.la $(top_builddir)/libkcal/libkcal.la
 
 libkabc_slox_la_SOURCES = kabcresourceslox.cpp kabcresourcesloxconfig.cpp \
                           kabcsloxprefs.kcfgc
-libkabc_slox_la_LDFLAGS = $(all_libraries)
+libkabc_slox_la_LDFLAGS = $(all_libraries) -no-undefined
 libkabc_slox_la_LIBADD  = libkslox.la  -lkabc
 
 
--- origsrc/kdepim-3.5.9/networkstatus/Makefile.am	2005-09-10 03:24:02.000000000 -0500
+++ src/kdepim-3.5.9/networkstatus/Makefile.am	2008-05-02 11:39:37.640625000 -0500
@@ -18,8 +18,8 @@
 kded_networkstatus_la_SOURCES = networkstatus.cpp networkstatus.skel \
 	clientiface.skel serviceiface.skel network.cpp
 #kclientiface.skel serviceiface.skel
-kded_networkstatus_la_LIBADD = $(LIB_KDECORE) $(LIB_KIO) ./libnetworkstatus.la
-kded_networkstatus_la_LDFLAGS = $(all_libraries) -module -avoid-version 
+kded_networkstatus_la_LIBADD = $(LIB_KDECORE) $(LIB_KIO) $(LIB_KDED) libnetworkstatus.la
+kded_networkstatus_la_LDFLAGS = $(all_libraries) $(KDE_PLUGIN)
 
 servicesdir = $(kde_servicesdir)/kded
 services_DATA = networkstatus.desktop
--- origsrc/kdepim-3.5.9/wizards/Makefile.am	2008-02-13 03:39:28.000000000 -0600
+++ src/kdepim-3.5.9/wizards/Makefile.am	2008-05-03 23:54:47.140625000 -0500
@@ -24,7 +24,7 @@
 groupwarewizard_SOURCES = groupwarewizard.cpp main.cpp overviewpage.cpp
 
 # Kolab
-libkolabwizard_la_LDFLAGS = -avoid-version -no-undefined $(all_libraries)
+libkolabwizard_la_LDFLAGS = $(KDE_PLUGIN) $(all_libraries)
 libkolabwizard_la_LIBADD = $(top_builddir)/kresources/kolab/kcal/libkcalkolab.la \
                            $(top_builddir)/kresources/kolab/kabc/libkabckolab.la \
                            $(top_builddir)/kresources/kolab/knotes/libknoteskolab.la \
@@ -40,7 +40,7 @@
 kolabwizard_SOURCES = kolabmain.cpp
 
 # eGroupware
-libegroupwarewizard_la_LDFLAGS = -avoid-version -no-undefined $(all_libraries)
+libegroupwarewizard_la_LDFLAGS = $(KDE_PLUGIN) $(all_libraries)
 libegroupwarewizard_la_SOURCES = egroupwarewizard.cpp egroupwareconfig.kcfgc
 libegroupwarewizard_la_LIBADD = $(top_builddir)/kresources/egroupware/libkabc_xmlrpc.la \
                                 $(top_builddir)/kresources/egroupware/libkcal_xmlrpc.la \
@@ -54,7 +54,7 @@
 egroupwarewizard_SOURCES = egroupwaremain.cpp
 
 # SLOX
-libsloxwizard_la_LDFLAGS = -avoid-version -no-undefined $(all_libraries)
+libsloxwizard_la_LDFLAGS = $(KDE_PLUGIN) $(all_libraries)
 libsloxwizard_la_SOURCES = sloxwizard.cpp sloxconfig.kcfgc
 libsloxwizard_la_LIBADD = $(top_builddir)/kresources/slox/libkcal_slox.la \
                           $(top_builddir)/kresources/slox/libkabc_slox.la \
@@ -65,7 +65,7 @@
 sloxwizard_SOURCES = sloxmain.cpp
 
 # GroupWise
-libgroupwisewizard_la_LDFLAGS = -avoid-version -no-undefined $(all_libraries)
+libgroupwisewizard_la_LDFLAGS = $(KDE_PLUGIN) $(all_libraries)
 libgroupwisewizard_la_SOURCES = groupwisewizard.cpp groupwiseconfig.kcfgc \
   kmailchanges.cpp
 libgroupwisewizard_la_LIBADD = $(top_builddir)/kresources/groupwise/libkcal_groupwise.la \
@@ -81,7 +81,7 @@
 groupwisewizard_SOURCES = groupwisemain.cpp
 
 # Exchange
-libexchangewizard_la_LDFLAGS = -avoid-version -no-undefined $(all_libraries)
+libexchangewizard_la_LDFLAGS = $(KDE_PLUGIN) $(all_libraries)
 libexchangewizard_la_SOURCES = exchangewizard.cpp
 libexchangewizard_la_LIBADD = $(top_builddir)/kresources/newexchange/libkcal_newexchange.la \
                               $(top_builddir)/kresources/newexchange/libkabc_newexchange.la \
@@ -92,7 +92,7 @@
 exchangewizard_SOURCES = exchangemain.cpp
 
 # Scalix
-libscalixwizard_la_LDFLAGS = -avoid-version -no-undefined $(all_libraries)
+libscalixwizard_la_LDFLAGS = $(KDE_PLUGIN) $(all_libraries)
 libscalixwizard_la_LIBADD = $(top_builddir)/kresources/scalix/kcal/libkcalscalix.la \
                            $(top_builddir)/kresources/scalix/kabc/libkabcscalix.la \
                            $(top_builddir)/libkcal/libkcal.la \
